scripts/internal/convert_readme.py

```diff
@@ -44,7 +44,11 @@ def main():
         data = f.read()
     data = re.sub(r".. raw:: html\n+\s+<div align[\s\S]*?/div>", summary, data)
     data = re.sub(r"Sponsors\n========[\s\S]*?Example usages", funding, data)
-    print(data)
+    if len(sys.argv) > 2:
+        with open(sys.argv[2], "wb") as f:
+            f.write(data.encode("utf8"))
```

Comment:
Couldn't you simply do `data.encode("utf8")` from setup.py and avoid editing this file?

Reply:
@mayeut this one is [missing](https://github.com/ddelange/psutil/actions/runs/3442949410/jobs/5744010265#step:6:178) `import sys`

Reply:
It wasn't missing in my commit (since `sys` was already used to read the first argument) so I don't know what went wrong on master.

Reply:
https://github.com/giampaolo/psutil/commit/ad8426c463b24b4b39b722b4d28949f2080182fc

Reply:
That's my fault actually. I manually edited @mayeut's PR to get rid of some parts, then merged it.